src/components/Hero/Hero.stories.tsx

```diff
@@ -273,15 +380,45 @@ export const Campaign: Story = {
           imageProps={imageProps}
           subHeaderText={otherSubHeaderText}
         />
+        <Source
+          code={`
+<Hero
+  backgroundImageSrc="//loremflickr.com/1600/800/kitten"
```

Comment:
the background image is a bit too gruesome to use...

![Screen Shot 2024-04-05 at 11 56 31 AM](https://github.com/NYPL/nypl-design-system/assets/1280564/df62e64a-74dc-4beb-8f39-0b37fe332d7c)